charts/gateway/production-values.yaml

```diff
@@ -354,6 +357,49 @@ config:
         # - Built-in services
         tls:
           enabled: false
+  redis:
+    # enable or disable redis integration
+    # please uncomment the 3 redis properties in config.systemProperties
+    enabled: false
+    # Use an existing secret for Redis configuration
+    # redis configuration must have a key called redis.properties
+    # existingConfigSecret: redis-properties
+    subChart:
+      # If the subChart is enabled, configuration from redis in the subCharts section
+      # will need to match the auth and tls settings here. This is true by default so no
+      # changes will be required
+      # Using Redis as a subChart is not recommended in production, please use your own Redis for this integration.
+      enabled: false
+  # groupName:
+    auth:
+      enabled: false
+    # username: username
+      password:
+        encoded: false
+        value: "mypassword"
+    # In standalone mode auth and ssl/tls is not available
+    # this is for testing only and is not supported in
+    # production environments
+    sentinel:
+      enabled: true
+      masterSet: mymaster
+      # If the subChart is not enabled sentinel nodes need to be set
+      nodes:
+      - <host>:<port>
+      - <host>:<port>
+      - <host>:<port>
+    standalone: {}
+    #  host: redis-standalone
+    #  port: 6379
+    tls:
+      enabled: true
```

Comment:
We should update here to be consistent of the default value in Readme

Reply:
Updated